Mage.Tests/src/test/java/org/mage/test/serverside/base/impl/CardTestPlayerAPIImpl.java

```diff
@@ -2159,6 +2159,82 @@ public GameView getGameView(Player player) {
         return GameSessionPlayer.prepareGameView(currentGame, player.getId(), null);
     }
 
+    public OnTurn onTurn(int turnNum) {
+        return new OnTurn(turnNum);
+    }
+
+    public class OnTurn {
```

Comment:
In my opinion it'd be best to add new files for new classes. CardTestPlayerAPIImpl is already way, way too long at over 2,000 lines so it'd be good to use new files whenever possible.